src/vs/platform/windows/electron-main/windowsService.ts

```diff
@@ -371,6 +371,10 @@ export class WindowsService implements IWindowsService, IURLHandler, IDisposable
 
 	}
 
+	async sendToSharedProcess(channel: string, ...args: any[]): Promise<void> {
```

Comment:
The whole point of the IPC abstraction is that we stop using strings. This should be `reportWorkspaceStats(workspace: IWorkspace)` instead.